client/src/browse/maps/Map.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from "react";
+import GoogleMapReact from "google-map-react";
+
+import MapMarker from "./MapMarker.js";
+require("dotenv").config();
+
```

Comment:
Does this work for you? I had to not use dotenv in my code.

Reply:
@7socks I used env-cmd yesterday, instead of importing a file it just runs a file in the npm script which in turn allows all downstream processes access to those env's defined in the file. I think the npm script "build:staging" uses the env-cmd if you want to look.

Reply:
Removed dotenv, will replace with env-cmd in "integration" PR